lib/asciidoctor-revealjs/highlightjs.rb

```diff
@@ -50,8 +52,317 @@ def docinfo location, doc, opts
           else
             theme_href = "#{revealjsdir}/lib/css/monokai.css"
           end
-          %(<link rel="stylesheet" href="#{theme_href}"#{opts[:self_closing_tag_slash]}>)
+          base_url = doc.attr 'highlightjsdir', %(#{opts[:cdn_base_url]}/highlight.js/#{HIGHLIGHT_JS_VERSION})
+          %(<link rel="stylesheet" href="#{theme_href}"#{opts[:self_closing_tag_slash]}>
+<script src="#{base_url}/highlight.min.js"></script>
+#{(doc.attr? 'highlightjs-languages') ? ((doc.attr 'highlightjs-languages').split ',').map {|lang| %[<script src="#{base_url}/languages/#{lang.lstrip}.min.js"></script>\n] }.join : ''}<script>#{HIGHLIGHT_PLUGIN_SOURCE}
```

Comment:
I would add whitespace here, it's confusing that the opening `<script>` tag doesn't depend on the presence of the `highlightjs-languages` attribute.
```suggestion
#{(doc.attr? 'highlightjs-languages') ? ((doc.attr 'highlightjs-languages').split ',').map {|lang| %[<script src="#{base_url}/languages/#{lang.lstrip}.min.js"></script>\n] }.join : ''}
<script>
#{HIGHLIGHT_PLUGIN_SOURCE}
```

Reply:
> I would add whitespace here, it's confusing that the opening <script> tag doesn't depend on the presence of the highlightjs-languages attribute.

I took the code from the original class: https://github.com/asciidoctor/asciidoctor/blob/b075a886d599e66841cd26416c23f6fb9e0a9c92/lib/asciidoctor/syntax_highlighter/highlightjs.rb#L23

But it's true that this line is dense!

Reply:
Oh I see, it was all on one line because it's all in a big `%( ... )` block.

Well if it doesn't break anything, I would definitely multi-line this big thing.